specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -10033,6 +10041,7 @@
     },
     "IgnoreResetRequirementsParameter": {
       "name": "ignoreResetRequirements",
+      "x-ms-client-name": "bypassIndexerResetRequirementForCache",
```

Comment:
Can we use `skip` instead of `bypass`?  That naming pattern is more common in .NET.

Reply:
Seems like a good suggestion to me.
@jadrefke, you OK with it?

Reply:
yeah, the name of this parameter is largely the preference of the SDK team. We've already decided on the parameter name service side

Reply:
our documentation will be using "ignoreResetRequirements" to describe the feature though

Reply:
Changed to `skipIndexerResetRequirementForCache`.